roles/confluent.test/molecule/Dockerfile-debian.j2

```diff
@@ -12,26 +12,30 @@ RUN apt-get update && \
       unzip \
       curl
 
-RUN wget -qO - https://packages.confluent.io/deb/5.5/archive.key | sudo apt-key add - && \
-    add-apt-repository "deb [arch=amd64] https://packages.confluent.io/deb/5.5 stable main" && \
+{% set PACKAGE_VER = lookup('env', 'VERSION') | default('5.5.7', true) %}
+{% set REPO_VER = lookup('env', 'VERSION') | regex_replace('^([0-9])\\.([0-9]*).*', '\\1.\\2') | default('5.5', true) %}
```

Comment:
This is ok. A better way would be to use PACKAGE_VER. As you are sure at his variable holds the version. You may not need default case here

Reply:
Sorry @nsharma-git, I don't understand this. 
`VERSION` is an env variable that'll be set by the user (or by script reading extra vars in molecule run) 
PACKAGE_VER resolves it, by assuring that it has a default value. This default value will be diff for diff .x branches. 
REPO_VER picks out the x.y from x.y.z (VERSION) or defaults to 5.5 (which'll again be diff for diff .x branch)

Reply:
Hi @nsharma-git, just bumping this.

Reply:
Do you mean something like this? 
{% set DEFAULT_PACKAGE_VER = 5.5.7 %}
{% set DEFAULT_REPO_VER = DEFAULT_PACKAGE_VER | regex_replace('^([0-9])\\.([0-9]*).*', '\\1.\\2') %}
{% set PACKAGE_VER = lookup('env', 'VERSION') | default(DEFAULT_PACKAGE_VER, true) %}
{% set REPO_VER = lookup('env', 'VERSION') | regex_replace('^([0-9])\\.([0-9]*).*', '\\1.\\2') | default(DEFAULT_REPO_VER, true) %}

Reply:
I was thinking somethig like this - 
```
{% set PACKAGE_VER = lookup('env', 'VERSION') | default('5.5.7', true) %}
{% set REPO_VER =$PACKAGE_VER | regex_replace('^([0-9])\\.([0-9]*).*', '\\1.\\2')%}
```

Reply:
Yep, thanks. My bad. This is more elegant.